app/quest-boost/claim/[boostId]/page.tsx

```diff
@@ -222,6 +234,28 @@ export default function Page({ params }: BoostQuestPageProps) {
           />
         </div>
       ) : null}
+      <ModalMessage
+        open={modalTxOpen}
+        title="Your NFT is on it's way !"
+        closeModal={() => {
+          setModalTxOpen(false);
+          router.back();
```

Comment:
Sorry I should have said it the first time but it makes more sense to make the user go to `/quest-boost` instead of `.back()` because back makes him go to the quest boost he just opened